.github/workflows/test-go.yaml

```diff
@@ -43,9 +48,15 @@ jobs:
         export PATH=$PATH:~/go/bin
         make generate-go
 
+    - name: Set Go race setting
+      if: github.event.schedule != '0 4 * * *'
```

Comment:
Is it just me, or does this seem backwards? Shouldn't we enable race detection when the schedule matches the cron above, not when it doesn't? Apologies for the double negatives.

Reply:
You are right. Good catch :)